### ISSUE TITLE
Is rack0.hpp up to date?

### ISSUE THREAD

**Issue by Rcomian** [VCVRack/Rack#1244]
Might well be something I'm doing, but when porting a module to v1, the #include "rack0.hpp" is failing with the error:

```
c++  -Wsuggest-override -std=c++11 -fPIC -I../../include -I../../dep/include -MMD -MP -g -O3 -march=nocona -funsafe-math-optimizations -Wall -Wextra -Wno-unused-parameter -DARCH_LIN -c -o build/src/GVerbWidget.cpp.o src/GVerbWidget.cpp
In file included from src/GVerbWidget.hpp:1,
                 from src/GVerbWidget.cpp:1:
../../include/rack0.hpp: In function ‘TParamWidget* rack::createParam(rack::math::Vec, rack::engine::Module*, int, float, float, float)’:
../../include/rack0.hpp:123:27: error: ‘__gnu_cxx::__alloc_traits<std::allocator<rack::engine::Param>, rack::engine::Param>::value_type’ {aka ‘struct rack::engine::Param’} has no member named ‘config’
   module->params[paramId].config(minValue, maxValue, defaultValue);
                           ^~~~~~
make: *** [../../compile.mk:63: build/src/GVerbWidget.cpp.o] Error 1
```

There doesn't seem to be a `.config` available on the param class anymore and I'm not sure of an obvious replacement.

**Comment by AndrewBelt** [comment]
rack0.hpp is probably still fine. You need to change your method call according to
https://community.vcvrack.com/t/rack-v1-development-blog/1149/374

**Comment by Rcomian** [comment]
Sorry bit more context. I'm not actually compiling any modules at this point, just the skeleton plugin.

![image](https://user-images.githubusercontent.com/684678/55994270-21ee3b00-5ca9-11e9-82e8-bc28c8cc73b9.png)


This is the failing code in `rack0.hpp`

**Comment by AndrewBelt** [comment]
Ah. Fixed dbb4d67
